vj4/db.py

```diff
@@ -8,6 +8,7 @@
 options.define('db_name', default='test', help='Database name.')
 options.define('db_username', default='', help='Database username.')
 options.define('db_password', default='', help='Database password.')
+options.define('db_auth', default='', help='Database name which stores the user’s credential.')
```

Comment:
`’` is not ascii character.

Also see the official description:
https://docs.mongodb.com/manual/reference/connection-string/#urioption.authSource